internal/provider/iam/resource_group_iam_policy_test.go

```diff
@@ -14,6 +14,8 @@ import (
 )
 
 func TestAccGroupIamPolicyResource(t *testing.T) {
+	t.Skip("TODO Fix this test")
```

Comment:
Without adding too much overhead/scope, is it possible to run these using the old method until the new way is completed? I don't imagine we will be introducing a regression anytime soon but just want to be safe with these workflows.

Reply:
If the work to add in these tests using the new methods is already underway then you can disregard this comment 😄

Reply:
Unfortunately the old way has been broken for over a year 😢 . This is because the scope of all the tests exceeded 6 hours of execution, which is the most the GH runners will go. Ordinarily I'd agree with you, and I don't like skipping tests, but I simply can't fix every broken test for all products at a first pass. I've done the ones I could, but some are trickier than others.

Reply:
By "old way" I was thinking more so of the old GH action that was replaced in this PR. I can understand that having redundant actions might make things confusing though so as long as we have a path forward I think this `skip` makes sense.

Reply:
Ah I see, yes sorry IAM is the only tests that actually work I forgot that. I am happy to work with you on getting them going if your willing? Will continue chat via slack.